libs/wingsdk/src/std/node.ts

```diff
@@ -89,6 +89,14 @@ export class Node {
    */
   public icon?: string;
 
+  /**
+   * The default view of this resource in the UI. By default, nodes are collapsed,
+   * so set this to `true` if you want the node to be expanded by default.
+   *
+   * @default false
+   */
+  public expanded?: boolean;
```

Comment:
I think it will be a very strange behavior if external libs with custom resources will have the ability to change the default expanded state of a node.
This of the following use case:
 - loading the app, resource A is expanded by default.
 - The user collapse resource A since they would like to focus on a different area in their app.
 - User changes the code, the console reloads - resource A is expanded again.
 
 I don't like it. If we are trying to handle a case where my app has only 1 resource and it looks strange being collapsed. lets handle this use case in the Console.